fieldservice/README.rst

```diff
@@ -85,6 +85,7 @@ Contributors
 * Wolfgang Hall <whall@opensourceintegrators.com>
 * Maxime Chambreuil <mchambreuil@opensourceintegrators.com>
 * Steve Campbell <scampbell@opensourceintegrators.com>
+* Bhavesh Odedra <bodedra@opensourceintegrators.com>
```

Comment:
@bodedra No need to change this file. It is updated automatically from the readme folder.